shared/app/global-errors/index.desktop.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import React, {Component} from 'react'
 import logger from '../../logger'
-import {Box, Text, Icon, HOCTimers} from '../../common-adapters'
+import {Box, Text, Icon} from '../../common-adapters'
+import {type TimerProps, default as HOCTimers} from '../../common-adapters/hoc-timers'
```

Comment:
you can do it like the react line above
import HOCTimers, {type TimerProps}

Reply:
Thanks, done